xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/table/livetabledate.js

```diff
@@ -0,0 +1,90 @@
+require.config({
+  paths: {
+    'moment': "$services.webjars.url('momentjs', 'moment.js')",
+    'jdateformatparser': "$services.webjars.url('org.webjars.bower:moment-jdateformatparser', 'moment-jdateformatparser.js')",
+    'daterangepicker': "$services.webjars.url('bootstrap-daterangepicker', 'js/bootstrap-daterangepicker.js')"
+  },
+  shim: {
+    'moment': ['jdateformatparser'],
+    'daterangepicker': ['jquery', 'bootstrap', 'moment']
+  }
+});
+
+require(['jquery', 'moment', 'daterangepicker', 'xwiki-events-bridge'],
+  function($, moment) {
+    var bindInputs = function(bind) {
+      $(bind).find('input[data-type="date"]').each(function(i, element) {
+        var input = $(element);
+        var hidden = input.prev('input[type="hidden"]');
+        var dateFormat = moment().toMomentFormatString(input.attr('data-dateformat'));
+        var livetable = input.closest('.xwiki-livetable');
```

Comment:
I think you already have a reference to the live table element in ``bind`` variable above. Depends whether you think ``bindInputs`` might be called on something other than a live table. Your choice.